### DISCUSSION TITLE
H&M Large scale

### DISCUSSION THREAD

**Discussion by NoAce4U** [Vdauphin/HeartsAndMinds#1166] [Question and answer]
We have a server running a large-scale 128 player version of H&M. Everything is going pretty smooth. Still need to iron out some of the mod issues. 

Any suggestions for performance you can throw at me? I've noticed after a bit AI decides to not want to spawn. Not sure where that's coming from just yet.

**Comment by Vdauphin** [comment]
Hello,

> Any suggestions for performance you can throw at me?

Nothing, I saw some people running it with 100 people. I just notice they choose to have an hideout per city so they can have more fun. From my point of view just increase the IA and ACE medical difficulty. May be tweak the number of IED so it is more challenging. But obviously, don't increase the number of IA.

>  I've noticed after a bit AI decides to not want to spawn. Not sure where that's coming from just yet.

Never saw it. May be your mod list or mission change just fuck up the mission. Also consider to restart at least every 3 days your server. Because CBA stop running after 7 days: https://github.com/CBATeam/CBA_A3/pull/1320

Cheers
